packages/shorebird_cli/lib/src/artifact_builder.dart

```diff
@@ -338,7 +338,10 @@ class ArtifactBuilder {
       final stdout = stdoutLines.join('\n');
       final exitCode = await buildProcess.exitCode;
       if (exitCode != ExitCode.success.code) {
-        throw ArtifactBuildException('Failed to build: $stderr');
+        throw ArtifactBuildException('''
+Failed to build
+stdout: $stdout
```

Comment:
stdout occasionally contains useful failure messages that are not included in stderr